cg_hermes/config/balsamic.py

```diff
@@ -188,6 +188,10 @@
     "dellycnv.cov.gz": ["rd-delly", "clinical-rd-delly"],
     "tumor.vcf2cytosure.cgh": ["cgh-tumor", "cnv-somatic-cgh-tumor"],
     "normal.vcf2cytosure.cgh": ["cgh-normal", "cnv-somatic-cgh-normal"],
+    "cov.bed.gz": ["cov", "gens-bed", "cnv-gens-bed"],
+    "cov.bed.gz.tbi": ["cov", "gens-bed", "cnv-gens-bed-index"],
+    "baf.bed.gz": ["baf", "gens-bed", "cnv-gens-bed"],
+    "baf.bed.gz.tbi": ["baf", "gens-bed", "cnv-gens-bed-index"],
```

Comment:
Have I understood it correctly that this is the file and tags from the balsamic `case.hk.yaml` ? And that the frozenset tags below is where they are translated into housekeeper tags?

Reply:
Exactly! Some time ago I added this dictionary to make it easier to link the actual files and the balsamic tags